packages/fxa-settings/src/pages/ResetPasswordRedesign/ConfirmResetPassword/index.tsx

```diff
@@ -47,21 +47,20 @@ const ConfirmResetPassword = ({
 
   return (
     <AppLayout>
-      <FtlMsg id="confirm-reset-password-otp-flow-name">
-        <p className="text-start text-grey-400 text-sm">Reset password</p>
+      <FtlMsg id="password-reset-flow-heading">
+        <p className="text-start text-grey-400 text-sm">Reset your password</p>
```

Comment:
Discussed on Slack that it might make more sense to have "Reset your password" as `h1` and "Check your email" as `h2` (considering that "check your email" is a sub-task of resetting the password), but our `CardHeader` component is currently configured as `h1`. Will leave as a `<p>` element for now, to be reviewed at a later time when we update the CardHeader component.